packages/sheets-ui/src/controllers/render-controllers/header-resize.render-controller.ts

```diff
@@ -492,19 +490,63 @@ export class HeaderResizeRenderController extends Disposable implements IRenderM
         this.disposeWithMe(
             toDisposable(
```

Comment:
`toDisposable` can be removed here as `disposeWithMe` can handle `Subscription`.